cmd/launcher/main.go

```diff
@@ -34,13 +37,13 @@ func runLaunch() error {
 		return err
 	}
 
-	defaultProcessType := defaultProcessType(api.MustParse(platformAPI), md)
+	defaultProcessType := defaultProcessType(api.MustParse(platform.API()), md)
 
 	launcher := &launch.Launcher{
 		DefaultProcessType: defaultProcessType,
 		LayersDir:          cmd.EnvOrDefault(cmd.EnvLayersDir, cmd.DefaultLayersDir),
 		AppDir:             cmd.EnvOrDefault(cmd.EnvAppDir, cmd.DefaultAppDir),
-		PlatformAPI:        api.MustParse(platformAPI),
+		PlatformAPI:        api.MustParse(platform.API()),
```

Comment:
Do you think we'll be able to remove `api.MustParse` in the future? Can the new `platform.API()` return `api.Version`? I see the constructor already has `api.MustParse(apiStr)`.

Reply:
I think it's a good idea but I'm not sure if we should do it as part of this PR.
We're using a string that is representing an api all over the code. Maybe we should dedicate a PR to change it.

Reply:
I like the idea of doing this in a separate PR.